src/client-utils/cf-images-utils.ts

```diff
@@ -30,5 +30,5 @@ export function getEdgeUrl(src: string, { name, ...variantParams }: Omit<EdgeUrl
   if (name.includes('.')) name = name.split('.').slice(0, -1).join('.') + '.jpeg';
   else name = name + '.jpeg';
 
-  return [env.NEXT_PUBLIC_IMAGE_LOCATION, src, params.toString(), name].join('/');
+  return [env.NEXT_PUBLIC_IMAGE_LOCATION, src, params.toString(), name].filter(Boolean).join('/');
```

Comment:
perhaps better use `isDefined`?

Reply:
In this case it's better to use `.filter(Boolean)`, cause we need to check for any falsy value like empty string and strip it out from the components of the url